src/main/java/core/strategy/FruitStrategy.java

```diff
@@ -0,0 +1,8 @@
+package core.strategy;
+
+import core.model.Fruit;
+import core.storage.DataBase;
+
+public interface FruitStrategy {
```

Comment:
I would suggest creating OperationHandler get(data) method for choosing right operation based on the data received. Also, execute() method we better place in the OperationHandler/Operation interface